README.md

```diff
@@ -44,6 +44,11 @@ For convenience purposes, the currently available Editor's Draft of subspecifica
             <td>Modbus</td>
             <td><a href="https://w3c.github.io/wot-binding-templates/bindings/protocols/modbus/index.html">Link</a></td>
         </tr>
+        <tr>
+            <td>BACnet</td>
+            <td>BACnet</td>
```

Comment:
Shouldn't this be "**B**uilding **A**utomation and **C**ontrol **Net**works"

Reply:
Yes, BACnet stands for "Building Automation and Control Networks" but is written "BACnet" and not "BACNet".

https://www.ashrae.org/technical-resources/technical-faqs/question-51-what-is-bacnet